
[2016-07-07 09:53:29] testing.ERROR: exception 'ErrorException' with message 'Undefined property: App\Controllers\Demo::$method' in /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php:281
Stack trace:
#0 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(281): Nova\Exception\Handler->handleError(8, 'Undefined prope...', '/var/www/sites/...', 281, Array)
#1 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(263): Nova\Routing\Controller->setupDefaultView()
#2 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(94): Nova\Routing\Controller->callAction('test', Array)
#3 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(63): Nova\Routing\ControllerDispatcher->call(Object(App\Controllers\Demo), Object(Nova\Routing\Route), 'test')
#4 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(963): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Controllers...', 'test')
#5 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}('param1', 'param2', 'param3', 'param4/param5/p...')
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/Route.php(113): call_user_func_array(Object(Closure), Array)
#7 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1049): Nova\Routing\Route->run(Object(Nova\Http\Request))
#8 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#9 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#16 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#17 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#18 {main} [] []
[2016-07-07 09:55:22] testing.ERROR: exception 'BadMethodCallException' with message 'Method [getParameters] does not exist.' in /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php:387
Stack trace:
#0 /var/www/sites/novatesting/app/Controllers/Demo.php(60): Nova\Routing\Controller->__call('getParameters', Array)
#1 /var/www/sites/novatesting/app/Controllers/Demo.php(60): App\Controllers\Demo->getParameters()
#2 [internal function]: App\Controllers\Demo->test('param1', 'param2', 'param3', 'param4/param5/p...')
#3 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(276): call_user_func_array(Array, Array)
#4 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(94): Nova\Routing\Controller->callAction('test', Array)
#5 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(63): Nova\Routing\ControllerDispatcher->call(Object(App\Controllers\Demo), Object(Nova\Routing\Route), 'test')
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(963): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Controllers...', 'test')
#7 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}('param1', 'param2', 'param3', 'param4/param5/p...')
#8 /var/www/sites/packages/novaframework/new-system/src/Routing/Route.php(113): call_user_func_array(Object(Closure), Array)
#9 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1049): Nova\Routing\Route->run(Object(Nova\Http\Request))
#10 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#11 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#12 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#13 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#16 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#17 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#18 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#19 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#20 {main} [] []
[2016-07-07 09:56:18] testing.ERROR: exception 'BadMethodCallException' with message 'Method [prepareResponse] does not exist.' in /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php:387
Stack trace:
#0 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(278): Nova\Routing\Controller->__call('prepareResponse', Array)
#1 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(278): App\Controllers\Demo->prepareResponse(Object(Nova\View\View))
#2 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(94): Nova\Routing\Controller->callAction('test', Array)
#3 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(63): Nova\Routing\ControllerDispatcher->call(Object(App\Controllers\Demo), Object(Nova\Routing\Route), 'test')
#4 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(963): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Controllers...', 'test')
#5 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}('param1', 'param2', 'param3', 'param4/param5/p...')
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/Route.php(113): call_user_func_array(Object(Closure), Array)
#7 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1049): Nova\Routing\Route->run(Object(Nova\Http\Request))
#8 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#9 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#16 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#17 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#18 {main} [] []
[2016-07-07 09:56:19] testing.ERROR: exception 'BadMethodCallException' with message 'Method [prepareResponse] does not exist.' in /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php:387
Stack trace:
#0 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(278): Nova\Routing\Controller->__call('prepareResponse', Array)
#1 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(278): App\Controllers\Demo->prepareResponse(Object(Nova\View\View))
#2 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(94): Nova\Routing\Controller->callAction('test', Array)
#3 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(63): Nova\Routing\ControllerDispatcher->call(Object(App\Controllers\Demo), Object(Nova\Routing\Route), 'test')
#4 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(963): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Controllers...', 'test')
#5 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}('param1', 'param2', 'param3', 'param4/param5/p...')
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/Route.php(113): call_user_func_array(Object(Closure), Array)
#7 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1049): Nova\Routing\Route->run(Object(Nova\Http\Request))
#8 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#9 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#16 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#17 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#18 {main} [] []
[2016-07-07 10:12:02] testing.ERROR: exception 'BadMethodCallException' with message 'Method [prepareResponse] does not exist.' in /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php:385
Stack trace:
#0 [internal function]: Nova\Routing\Controller->__call('prepareResponse', Array)
#1 [internal function]: App\Modules\Dashboard\Controllers\Admin\Dashboard->prepareResponse(Object(Nova\Routing\Route), Object(Nova\Http\Request), Object(Nova\Http\Response))
#2 /var/www/sites/packages/novaframework/new-system/src/Events/Dispatcher.php(195): call_user_func_array(Array, Array)
#3 /var/www/sites/packages/novaframework/new-system/src/Events/Dispatcher.php(151): Nova\Events\Dispatcher->fire('router.filter: ...', Array, true)
#4 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1474): Nova\Events\Dispatcher->until('router.filter: ...', Array)
#5 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1454): Nova\Routing\Router->callRouteFilter('@prepareRespons...', Array, Object(Nova\Routing\Route), Object(Nova\Http\Request), Object(Nova\Http\Response))
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1057): Nova\Routing\Router->callRouteAfter(Object(Nova\Routing\Route), Object(Nova\Http\Request), Object(Nova\Http\Response))
#7 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#8 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#9 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#11 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#16 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#17 {main} [] []
[2016-07-07 10:12:24] testing.ERROR: exception 'BadMethodCallException' with message 'Method [prepareResponse] does not exist.' in /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php:385
Stack trace:
#0 [internal function]: Nova\Routing\Controller->__call('prepareResponse', Array)
#1 [internal function]: App\Modules\Dashboard\Controllers\Admin\Dashboard->prepareResponse(Object(Nova\Routing\Route), Object(Nova\Http\Request), Object(Nova\Http\Response))
#2 /var/www/sites/packages/novaframework/new-system/src/Events/Dispatcher.php(195): call_user_func_array(Array, Array)
#3 /var/www/sites/packages/novaframework/new-system/src/Events/Dispatcher.php(151): Nova\Events\Dispatcher->fire('router.filter: ...', Array, true)
#4 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1474): Nova\Events\Dispatcher->until('router.filter: ...', Array)
#5 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1454): Nova\Routing\Router->callRouteFilter('@prepareRespons...', Array, Object(Nova\Routing\Route), Object(Nova\Http\Request), Object(Nova\Http\Response))
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1057): Nova\Routing\Router->callRouteAfter(Object(Nova\Routing\Route), Object(Nova\Http\Request), Object(Nova\Http\Response))
#7 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#8 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#9 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#11 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#16 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#17 {main} [] []
[2016-07-07 10:12:25] testing.ERROR: exception 'BadMethodCallException' with message 'Method [prepareResponse] does not exist.' in /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php:385
Stack trace:
#0 [internal function]: Nova\Routing\Controller->__call('prepareResponse', Array)
#1 [internal function]: App\Modules\Dashboard\Controllers\Admin\Dashboard->prepareResponse(Object(Nova\Routing\Route), Object(Nova\Http\Request), Object(Nova\Http\Response))
#2 /var/www/sites/packages/novaframework/new-system/src/Events/Dispatcher.php(195): call_user_func_array(Array, Array)
#3 /var/www/sites/packages/novaframework/new-system/src/Events/Dispatcher.php(151): Nova\Events\Dispatcher->fire('router.filter: ...', Array, true)
#4 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1474): Nova\Events\Dispatcher->until('router.filter: ...', Array)
#5 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1454): Nova\Routing\Router->callRouteFilter('@prepareRespons...', Array, Object(Nova\Routing\Route), Object(Nova\Http\Request), Object(Nova\Http\Response))
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1057): Nova\Routing\Router->callRouteAfter(Object(Nova\Routing\Route), Object(Nova\Http\Request), Object(Nova\Http\Response))
#7 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#8 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#9 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#11 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#16 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#17 {main} [] []
[2016-07-07 10:21:09] testing.ERROR: exception 'ErrorException' with message 'Argument 1 passed to Nova\Routing\Controller::prepareResponse() must be an instance of Symfony\Component\HttpFoundation\Response, instance of Nova\View\View given, called in /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php on line 270 and defined' in /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php:301
Stack trace:
#0 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(301): Nova\Exception\Handler->handleError(4096, 'Argument 1 pass...', '/var/www/sites/...', 301, Array)
#1 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(270): Nova\Routing\Controller->prepareResponse(Object(Nova\View\View))
#2 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(94): Nova\Routing\Controller->callAction('index', Array)
#3 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(63): Nova\Routing\ControllerDispatcher->call(Object(App\Controllers\Welcome), Object(Nova\Routing\Route), 'index')
#4 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(963): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Controllers...', 'index')
#5 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/Route.php(113): call_user_func_array(Object(Closure), Array)
#7 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1049): Nova\Routing\Route->run(Object(Nova\Http\Request))
#8 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#9 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#10 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#11 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#12 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#13 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#14 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#15 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#16 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#17 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#18 {main} [] []
[2016-07-07 10:26:00] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: csrfToken' in /var/www/sites/novatesting/app/Modules/Users/Views/Admin/Users/Index.php:94
Stack trace:
#0 /var/www/sites/novatesting/app/Modules/Users/Views/Admin/Users/Index.php(94): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 94, Array)
#1 /var/www/sites/packages/novaframework/new-system/src/View/View.php(105): require('/var/www/sites/...')
#2 /var/www/sites/packages/novaframework/new-system/src/View/View.php(79): Nova\View\View->render()
#3 /var/www/sites/packages/novaframework/new-system/src/View/View.php(132): Nova\View\View->fetch()
#4 /var/www/sites/packages/novaframework/new-system/src/View/View.php(98): Nova\View\View->gatherData()
#5 /var/www/sites/packages/novaframework/new-system/src/View/View.php(79): Nova\View\View->render()
#6 /var/www/sites/packages/novaframework/new-system/src/Http/Response.php(45): Nova\View\View->fetch()
#7 /var/www/sites/novatesting/vendor/symfony/http-foundation/Response.php(201): Nova\Http\Response->setContent(Object(Nova\View\View))
#8 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(318): Symfony\Component\HttpFoundation\Response->__construct(Object(Nova\View\View))
#9 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(270): Nova\Routing\Controller->prepareResponse(Object(Nova\View\View))
#10 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(94): Nova\Routing\Controller->callAction('index', Array)
#11 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(63): Nova\Routing\ControllerDispatcher->call(Object(App\Modules\Users\Controllers\Admin\Users), Object(Nova\Routing\Route), 'index')
#12 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(963): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Modules\\Use...', 'index')
#13 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#14 /var/www/sites/packages/novaframework/new-system/src/Routing/Route.php(113): call_user_func_array(Object(Closure), Array)
#15 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1049): Nova\Routing\Route->run(Object(Nova\Http\Request))
#16 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#17 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#18 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#19 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#20 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#21 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#22 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#23 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#24 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#25 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#26 {main} [] []
[2016-07-07 10:29:48] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: baseUri' in /var/www/sites/novatesting/app/Templates/AdminLte/backend.php:168
Stack trace:
#0 /var/www/sites/novatesting/app/Templates/AdminLte/backend.php(168): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 168, Array)
#1 /var/www/sites/packages/novaframework/new-system/src/View/View.php(105): require('/var/www/sites/...')
#2 /var/www/sites/packages/novaframework/new-system/src/View/View.php(79): Nova\View\View->render()
#3 /var/www/sites/packages/novaframework/new-system/src/Http/Response.php(45): Nova\View\View->fetch()
#4 /var/www/sites/novatesting/vendor/symfony/http-foundation/Response.php(201): Nova\Http\Response->setContent(Object(Nova\View\View))
#5 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(318): Symfony\Component\HttpFoundation\Response->__construct(Object(Nova\View\View))
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(270): Nova\Routing\Controller->prepareResponse(Object(Nova\View\View))
#7 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(94): Nova\Routing\Controller->callAction('index', Array)
#8 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(63): Nova\Routing\ControllerDispatcher->call(Object(App\Modules\Users\Controllers\Admin\Users), Object(Nova\Routing\Route), 'index')
#9 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(963): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Modules\\Use...', 'index')
#10 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#11 /var/www/sites/packages/novaframework/new-system/src/Routing/Route.php(113): call_user_func_array(Object(Closure), Array)
#12 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1049): Nova\Routing\Route->run(Object(Nova\Http\Request))
#13 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#14 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#15 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#16 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#17 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#18 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#19 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#20 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#21 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#22 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#23 {main} [] []
[2016-07-07 10:32:07] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: baseUri' in /var/www/sites/novatesting/app/Templates/AdminLte/backend.php:168
Stack trace:
#0 /var/www/sites/novatesting/app/Templates/AdminLte/backend.php(168): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 168, Array)
#1 /var/www/sites/packages/novaframework/new-system/src/View/View.php(105): require('/var/www/sites/...')
#2 /var/www/sites/packages/novaframework/new-system/src/View/View.php(79): Nova\View\View->render()
#3 /var/www/sites/packages/novaframework/new-system/src/Http/Response.php(45): Nova\View\View->fetch()
#4 /var/www/sites/novatesting/vendor/symfony/http-foundation/Response.php(201): Nova\Http\Response->setContent(Object(Nova\View\View))
#5 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(318): Symfony\Component\HttpFoundation\Response->__construct(Object(Nova\View\View))
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(270): Nova\Routing\Controller->prepareResponse(Object(Nova\View\View))
#7 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(94): Nova\Routing\Controller->callAction('index', Array)
#8 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(63): Nova\Routing\ControllerDispatcher->call(Object(App\Modules\Users\Controllers\Admin\Users), Object(Nova\Routing\Route), 'index')
#9 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(963): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Modules\\Use...', 'index')
#10 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#11 /var/www/sites/packages/novaframework/new-system/src/Routing/Route.php(113): call_user_func_array(Object(Closure), Array)
#12 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1049): Nova\Routing\Route->run(Object(Nova\Http\Request))
#13 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#14 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#15 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#16 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#17 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#18 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#19 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#20 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#21 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#22 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#23 {main} [] []
[2016-07-07 10:37:37] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: uri' in /var/www/sites/novatesting/app/Core/Controller.php:49
Stack trace:
#0 /var/www/sites/novatesting/app/Core/Controller.php(49): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 49, Array)
#1 [internal function]: App\Core\Controller->setupVariables(Object(Nova\Routing\Route), Object(Nova\Http\Request))
#2 /var/www/sites/packages/novaframework/new-system/src/Events/Dispatcher.php(195): call_user_func_array(Array, Array)
#3 /var/www/sites/packages/novaframework/new-system/src/Events/Dispatcher.php(151): Nova\Events\Dispatcher->fire('router.filter: ...', Array, true)
#4 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1474): Nova\Events\Dispatcher->until('router.filter: ...', Array)
#5 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(235): Nova\Routing\Router->callRouteFilter('@setupVariables', Array, Object(Nova\Routing\Route), Object(Nova\Http\Request))
#6 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(113): Nova\Routing\ControllerDispatcher->callFilter(Array, Object(Nova\Routing\Route), Object(Nova\Http\Request))
#7 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(57): Nova\Routing\ControllerDispatcher->before(Object(App\Modules\Users\Controllers\Admin\Users), Object(Nova\Routing\Route), Object(Nova\Http\Request), 'index')
#8 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(963): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Modules\\Use...', 'index')
#9 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#10 /var/www/sites/packages/novaframework/new-system/src/Routing/Route.php(113): call_user_func_array(Object(Closure), Array)
#11 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1049): Nova\Routing\Route->run(Object(Nova\Http\Request))
#12 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#13 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#14 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#15 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#16 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#17 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#18 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#19 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#20 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#21 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#22 {main} [] []
[2016-07-07 11:09:10] testing.ERROR: exception 'ErrorException' with message 'Undefined variable: csrfToken' in /var/www/sites/novatesting/app/Modules/Settings/Views/Admin/Settings/Index.php:159
Stack trace:
#0 /var/www/sites/novatesting/app/Modules/Settings/Views/Admin/Settings/Index.php(159): Nova\Exception\Handler->handleError(8, 'Undefined varia...', '/var/www/sites/...', 159, Array)
#1 /var/www/sites/packages/novaframework/new-system/src/View/View.php(105): require('/var/www/sites/...')
#2 /var/www/sites/packages/novaframework/new-system/src/View/View.php(79): Nova\View\View->render()
#3 /var/www/sites/packages/novaframework/new-system/src/View/View.php(132): Nova\View\View->fetch()
#4 /var/www/sites/packages/novaframework/new-system/src/View/View.php(98): Nova\View\View->gatherData()
#5 /var/www/sites/packages/novaframework/new-system/src/View/View.php(79): Nova\View\View->render()
#6 /var/www/sites/packages/novaframework/new-system/src/Http/Response.php(45): Nova\View\View->fetch()
#7 /var/www/sites/novatesting/vendor/symfony/http-foundation/Response.php(201): Nova\Http\Response->setContent(Object(Nova\View\View))
#8 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(305): Symfony\Component\HttpFoundation\Response->__construct(Object(Nova\View\View))
#9 /var/www/sites/packages/novaframework/new-system/src/Routing/Controller.php(289): Nova\Routing\Controller->prepareResponse(Object(Nova\View\View))
#10 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(94): Nova\Routing\Controller->callAction('index', Array)
#11 /var/www/sites/packages/novaframework/new-system/src/Routing/ControllerDispatcher.php(63): Nova\Routing\ControllerDispatcher->call(Object(App\Modules\Settings\Controllers\Admin\Settings), Object(Nova\Routing\Route), 'index')
#12 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(963): Nova\Routing\ControllerDispatcher->dispatch(Object(Nova\Routing\Route), Object(Nova\Http\Request), 'App\\Modules\\Set...', 'index')
#13 [internal function]: Nova\Routing\Router->Nova\Routing\{closure}()
#14 /var/www/sites/packages/novaframework/new-system/src/Routing/Route.php(113): call_user_func_array(Object(Closure), Array)
#15 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1049): Nova\Routing\Route->run(Object(Nova\Http\Request))
#16 /var/www/sites/packages/novaframework/new-system/src/Routing/Router.php(1005): Nova\Routing\Router->dispatchToRoute(Object(Nova\Http\Request))
#17 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(768): Nova\Routing\Router->dispatch(Object(Nova\Http\Request))
#18 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(742): Nova\Foundation\Application->dispatch(Object(Nova\Http\Request))
#19 /var/www/sites/novatesting/app/Extensions/Http/ContentGuard.php(61): Nova\Foundation\Application->handle(Object(Nova\Http\Request), 1, true)
#20 /var/www/sites/packages/novaframework/new-system/src/Session/Middleware.php(77): App\Extensions\Http\ContentGuard->handle(Object(Nova\Http\Request), 1, true)
#21 /var/www/sites/packages/novaframework/new-system/src/Cookie/Queue.php(49): Nova\Session\Middleware->handle(Object(Nova\Http\Request), 1, true)
#22 /var/www/sites/packages/novaframework/new-system/src/Cookie/Guard.php(54): Nova\Cookie\Queue->handle(Object(Nova\Http\Request), 1, true)
#23 /var/www/sites/novatesting/vendor/stack/builder/src/Stack/StackedHttpKernel.php(23): Nova\Cookie\Guard->handle(Object(Nova\Http\Request), 1, true)
#24 /var/www/sites/packages/novaframework/new-system/src/Foundation/Application.php(639): Stack\StackedHttpKernel->handle(Object(Nova\Http\Request))
#25 /var/www/sites/novatesting/public/index.php(54): Nova\Foundation\Application->run()
#26 {main} [] []
